src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -220,6 +228,13 @@ public void setDepartment(Department department) {
         public Optional<Department> getDepartment() {
             return Optional.ofNullable(department);
         }
+        public void setLeave(Leave leave) {
```

Comment:
The implementation of setLeave and getLeave is consistent with the existing setDepartment/getDepartment, which is good